src/components/FeeModal/FeeBreakdown.tsx

```diff
@@ -24,7 +25,44 @@ export const FeeBreakdown = ({ feeModel, inputAmountUsd }: FeeBreakdownProps) =>
     feeUsd: affiliateFeeAmountUsd,
     foxDiscountUsd,
     feeUsdBeforeDiscount,
+    feeBps,
   } = useAppSelector(state => selectCalculatedFees(state, { feeModel, inputAmountUsd }))
+  const appliedDiscountType = useAppSelector(state =>
+    selectAppliedDiscountType(state, { feeModel, inputAmountUsd }),
+  )
+  const greenColor = useColorModeValue('green.500', 'green.300')
+
+  const discountTypeTranslation = useMemo(() => {
+    return translate(`foxDiscounts.${appliedDiscountType}`)
+  }, [appliedDiscountType, translate])
+
+  const discountLabelTranslation = useMemo(() => {
+    return translate(`foxDiscounts.discountLabel`, { discountType: discountTypeTranslation })
+  }, [discountTypeTranslation, translate])
+
+  const totalAmount = useMemo(() => {
+    if (feeBps.isZero())
```

Comment:
Those `feeBps.isZero()` are the actual fixes, in the end in every flow we use `feeBps` which is not taking in account very small fees (0.001 bpsFloat  is resulting in 0 bps)

We start taking fees when bpsFloat is 0.01 and so bps is 1